typed-racket-lib/typed-racket/utils/mutated-vars.rkt

```diff
@@ -17,8 +19,18 @@
         (loop stx tbl)))
     (syntax-parse stx
       #:literal-sets (kernel-literals)
+      ;; let us care about custom syntax classes
+      [form
```

Comment:
Invalidate occurrence typing on what? Are we using this to track function calls as well as mutable variables?